packages/communication-react/package.json

```diff
@@ -45,7 +45,7 @@
     "uuid": "^8.1.0"
   },
   "peerDependencies": {
-    "@azure/communication-calling": "1.3.2-beta.1 || 1.3.2",
+    "@azure/communication-calling": "1.3.2-beta.1",
```

Comment:
Is this correct? 
Why did this remove the || stable dep?

Reply:
nope nice catch! I did a find and replace on `@azure/communication-calling": "1.4.1-beta.1` .. I guess another question is why is this not `1.4.1-beta.1`?

Reply:
Yes, the dep should be consistent across packages. This is strange

Reply:
I will keep as 1.3.2-beta.1 for this PR and then do a seperate PR that updates this straight into main 👍

Reply:
This might actually impact the beta release as the communication-react package should've had 1.4.1-beta.1 /:

Reply:
PR to main to fix this: https://github.com/Azure/communication-ui-library/pull/1594